scripts/script_utils.py

```diff
@@ -194,16 +207,16 @@ def load_rules_from_local(args) -> List[dict]:
             rule_data[rule_identifier] = rule
             rules.append(rule)
         else:
-            engine_logger.warning(
+            engine_logger.error(
                 f"Rule {rule.get('core_id')} not specified with "
-                "-r rule flag or duplicate rule. Skipping..."
+                "-r rule flag and in local directory or is a duplicate rule. Skipping..."
```

Comment:
I think skipping a rule in the local directory is somewhat typical and should actually be downgraded to `info`. Detecting a duplicate rule should be an error.  These 2 scenarios should be separate. Also, I don't see how this is detecting duplicate rules at all, but maybe I'm misunderstanding.